repository/kafka/docs/v1.0/release-notes.md

```diff
@@ -0,0 +1,18 @@
+# Release notes 
+
+## latest
```

Comment:
This should be v.1.0.0, no?

Reply:
good catch! updated and need to get https://jira.mesosphere.com/browse/DCOS-57399 to avoid that in future